src/register/ConfigurableRegistrationForm.jsx

```diff
@@ -54,13 +54,13 @@ const ConfigurableRegistrationForm = (props) => {
   });
 
   const handleOnChange = (event, countryValue = null) => {
-    const { name, type } = event.target;
+    const { name } = event.target;
     let value;
     if (countryValue) {
       value = { ...countryValue };
     } else {
       value = event.target.type === 'checkbox' ? event.target.checked : event.target.value;
-      if (type === 'checkbox') {
+      if (event.target.type === 'checkbox') {
```

Comment:
event.target.type i think we should destructure  it that would make more sense

Reply:
Yes, you are right! Destructuring is always best but in this use case. We are using this `handleOnChange` function for two types of fields, `text` (for country) and `check box` (for TOS, honor code, etc). 
By using the `onChange` method of the autosuggest field, we get the value only, not the event. means there is no `target.type` available. 
So we are left with two options here, either don't destructure the property that is not available for all use cases or pass hardcoded `type='text'` while calling this `handleOnChange` function. And from these two options, i thought the later one is better. Please let me know if you have any suggestions regarding this.